crypto/fipsmodule/cipher/cipher.c

```diff
@@ -353,8 +359,13 @@ int EVP_EncryptFinal_ex(EVP_CIPHER_CTX *ctx, uint8_t *out, int *out_len) {
 
 int EVP_DecryptUpdate(EVP_CIPHER_CTX *ctx, uint8_t *out, int *out_len,
                       const uint8_t *in, int in_len) {
-  int fix_len;
-  unsigned int b;
+  // Ciphers that use blocks may write up to |bl| extra bytes. Ensure the output
+  // does not overflow |*out_len|.
+  unsigned int b = ctx->cipher->block_size;
```

Comment:
Above we call this `bl`. Let's be consistent or align with bssl?

Reply:
Here, b seems correct because other places use `bl` as alias of `buf_len`.

Align with bssl because there are other inconsistent places.
```
  b = ctx->cipher->block_size;
  assert(b <= sizeof(ctx->buf));
  if (b == 1) {
    *out_len = 0;
    return 1;
  }

  bl = ctx->buf_len;
```

Reply:
Meh, I thought they used `bl` as an abbreviation of `block length`.
Vote for keep as-is.